runtime-config-linux.md

```diff
@@ -131,6 +131,58 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+The path to the cgroups can to be specified in the Spec via `cgroupsPath`.
+`cgroupsPath` is expected to be relative to the cgroups mount point.
+If not specified, cgroups will be created under '/' with the base name of the bundle.
```

Comment:
I think it's more reasonable / simpler to error out here rather than continuing and generate a path based on base name of the bundle. `cgroupsPath` should be a required data right ?